src/lib/chat/matrix/chat-message.ts

```diff
@@ -80,3 +80,15 @@ export function mapEventToAdminMessage(matrixMessage) {
     admin: adminData,
   };
 }
+
+export function mapToLiveRoomEvent(liveEvent) {
+  const { event } = liveEvent;
+
+  return {
+    id: event.event_id,
+    createdAt: event.origin_server_ts,
+    sender: {
+      userId: event.senderId,
+    },
+  };
```

Comment:
Are there different types of live events? Do we want our browser notifications to happen on EVERY type of event? I think we should probably add some kind of type field here so that our notification saga can notify us on the ones we actually want rather than have people get browser notifications for things that they know nothing about.

Reply:
Correct yes, without adding logic this would send browser notifications for any live event. Assuming (for now) that we only want to notify the user for a message event, I have updated the saga to handle this. Updated.